File: bs.js
function bubbleSort(arr) {
    var l = arr.length;
    for (var i = 0; i < l - 1; i++) {
        for (var j = 0; j < l - 1 - i; j++) {
            if (arr[j] > arr[j + 1]) {
                var temp = arr[j];
                arr[j] = arr[j + 1];
                arr[j + 1] = temp;
            }
        }
    }
    return arr;
}
var arr = [10, 7, 8, 9, 1, 5];
var sortedArr = bubbleSort(arr);
console.log("sorted array: " + sortedArr.join(" "));

File: bs.java
import java.util.Arrays;
class Main {
  static void bubbleSort(int array[]) 
  {
    int s = array.length;
      for (int i = 0; i < s - 1; i++)
      for (int j = 0; j < s - i - 1; j++)
        if (array[j] > array[j + 1]) 
        {
          int temp = array[j];
          array[j] = array[j + 1];
          array[j + 1] = temp;
        }
    }
  public static void main(String args[]) 
  {
    int[] arr = { 10, 7, 8, 9,1,5 };
    Main.bubbleSort(arr);
    System.out.println("Sorted Array:");
    System.out.println(Arrays.toString(arr));
  }
}